src/ore/src/tracing.rs

```diff
@@ -319,6 +321,50 @@ where
             .install_batch(opentelemetry::runtime::Tokio)
             .unwrap();
 
+        // Create our own error handler to:
+        //   1. Rate limit the number of error logs. By default the OTel library will emit
+        //      an enormous number of duplicate logs if any errors occur, one per batch
+        //      send attempt, until the error is resolved.
+        //   2. Log the errors via our tracing layer, so they are formatted consistently
+        //      with the rest of our logs, rather than the direct `eprintln` used by the
+        //      OTel library.
+        const OPENTELEMETRY_ERROR_MSG_BACKOFF_SECONDS: u64 = 30;
+        let last_log_in_epoch_seconds = AtomicU64::default();
+        opentelemetry::global::set_error_handler(move |err| {
+            let now = std::time::SystemTime::now()
+                .duration_since(std::time::SystemTime::UNIX_EPOCH)
+                .expect("Failed to get duration since Unix epoch")
+                .as_secs();
+            let last_log = last_log_in_epoch_seconds.load(Ordering::SeqCst);
+
+            if now.saturating_sub(last_log) >= OPENTELEMETRY_ERROR_MSG_BACKOFF_SECONDS {
+                if last_log_in_epoch_seconds
+                    .compare_exchange_weak(last_log, now, Ordering::SeqCst, Ordering::SeqCst)
```

Comment:
technically this can be `Relaxed` but this is fine too!

Reply:
I'll relax it -- I lean to SeqCst as a default since I've never learned what the other really mean. but, this is such a low stakes use case I wouldn't want us to spend any extra cpu cycles on it

Reply:
yes!